sphinx/source/style_properties.rst

```diff
@@ -1060,6 +1060,13 @@ These are used with the fixed layout.
     If True, the size of the fixed layout is shrunk vertically to match the
     bottom side of the bottommost child of the fixed.
 
+Note that children with relative positions or variable sizes can
+induce surprising behavior with Fixeds, when these properties are
+applied. This is because the children are first placed and sized as if
```

Comment:
I don't like the wording of this. Everything is surprising if you don't understand it. It makes sense to document this, but we don't need to say it's surprising.